src/pages/index.jsx

```diff
@@ -1,9 +1,103 @@
 import React, { Component } from "react"
 import Layout from "../components/layout/layout"
+import SecureTrading from "../stjs/st"
+import Helmet from "react-helmet/es/Helmet"
+
 class IndexPage extends Component {
+  componentDidMount() {
+    this.loadST()
+  }
+
+  loadST() {
+    this.config = {
```

Comment:
It would look nicer to move this to a separate file 😉

Reply:
true :) I did the same with animated card component